### DISCUSSION TITLE
Repeated Doses and no Equilibrium Reached

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1735] [Questions & Problems]
I altered the Itraconazole model available on GIT so that the patient was given a repeated dose either every 12 hours or every 6 hours(for about 6.5 days). In both cases, the amount of drug in the venous blood plasma just keeps increasing(almost linearly)! I expected it to level off. Any idea what's going on? 

Thanks,

DS  

<img width="731" alt="Screen Shot 2024-03-18 at 10 20 27 PM" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25123758/a07079da-dcdb-4482-8229-547934a7cce5">

<img width="755" alt="Screen Shot 2024-03-18 at 10 16 21 PM" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25123758/daa69fa7-f9c0-4233-8082-af7eac76a1c1">

**Comment by DonkeySaddle** [comment]
I am still confused by this. Does anyone have any idea what's going on? I just ran another Itraconazole simulation ("po 200 mg MD BID caps fed 1") in Mobi and it's not even coming close to achieving equilibrium.  

Any insight would be greatly appreciated! 

Thanks,

DS

<img width="865" alt="Itraconazole Venous Blood Plasma" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25123758/fb061d7f-c2bf-4514-b867-21d11a1aaadc">

**Reply by StephanSchaller** [reply]
Well, the point is that this is the expectet PK behaviour of Itracinazole. It is a highly accumulating drug. Check the original model file from the library.
